pkg/sql/sem/tree/backup.go

```diff
@@ -237,6 +248,12 @@ func (o *BackupOptions) Format(ctx *FmtCtx) {
 		ctx.WriteString("kms = ")
 		ctx.FormatNode(&o.EncryptionKMSURI)
 	}
+
+	if o.IncrementalStorage != nil {
```

Comment:
This looks like it wants a test in parser tests for round-tripping of the AST node, perhaps in pkg/sql/parser/testdata/backup_restore?

Reply:
done.